tests/sys/busy_wait/main.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2024 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief   Busy Wait loop Test Application
```

Comment:
In your PR you stated that you print the diff to the timer. Why not note that here?

Reply:
I added a sentence explaining that this can be used to determine `CPU_CYCLES_PER_LOOP`.